change/@ni-nimble-components-46b1fdfb-63b3-4a04-ac71-1124855a4e45.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix for Firefox caret disappearing issue",
+  "packageName": "@ni/nimble-components",
+  "email": "123377167+aagash-ni@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
Testing the change a bit I noticed this behavior:

![mouse down](https://github.com/ni/nimble/assets/1588923/9ffe298d-6ed1-4a58-b934-dbfd49493625)

When doing a mousedown on the italic button you see the bold button showing focus temporarily until you finish the mouse up.

I can also reproduce it on main so it might not be related to this specific change but could you double check and track in an issue if needed?